gr-pdu/lib/tags_to_pdu_impl.cc

```diff
@@ -156,8 +157,9 @@ void tags_to_pdu_impl<T>::publish_message()
     d_meta_dict = pmt::dict_add(
         d_meta_dict, metadata_keys::pdu_num(), pmt::from_uint64(d_burst_counter));
     if (d_wall_clock_time) {
-        double t_now((boost::get_system_time() - d_epoch).total_microseconds() /
-                     1000000.0);
+        double t_now(std::chrono::duration<double>(
+                         std::chrono::system_clock::now().time_since_epoch())
+                         .count());
```

Comment:
This assumes that the 'tick' value of the `duration` object is seconds. That makes sense and all, but is that guaranteed? I would have expected something like
```suggestion
                         std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now().time_since_epoch())
                         .count();
```

Reply:
OK I read that page but missed that detail. Thanks!